docs/controller-skills/callsigns.md

```diff
@@ -75,6 +75,17 @@ This page lists some of the less-known aircraft callsigns that are routinely use
 | ICE  | Ice                | Various  |
 | ASY  | Aussie             | RAAF     |
 
+### Military Formation
+These can be use as an extention of the above callsigns when formation flights take place eg. TEXGLD or MAKBLU. 
```

Comment:
Check grammar. It should be "These can be **used**". 

An important distinction is that not all of these are formation callsigns, some of them being tasking-specific. 

Additionally, as these are used for military activities, the use of these identifiers would likely be restricted to approved VSOAs only - at the moment there are no authorised NZ VSOAs able to use these. Suggest removal of this section as it is redundant.